src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -18,10 +18,11 @@
 
 public class Main implements IReporter {
   private static final String[] RESOURCES = new String[] {
-    "jquery-1.7.1.min.js", "testng-reports.css", "testng-reports.js",
-    "passed.png", "failed.png", "skipped.png", "navigator-bullet.png",
-    "bullet_point.png", "collapseall.gif"
```

Comment:
yeah good point, I've pushed up some changes for this.

I was unsure of the code style to follow, the file in src/main is 2 space formatted and the test is 4 and in a different package. I took a best guess at making it fit, let me know if it needs further change